src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -186,10 +186,10 @@ SDCPUThread::commitInstruction(std::shared_ptr<InflightInst> inst_ptr)
 {
     assert(inst_ptr->isComplete());
 
-    StaticInstPtr static_inst = inst_ptr->staticInst();
-
     DPRINTF(SDCPUInstEvent,
-             "Committing instruction (seq %d)\n", inst_ptr->seqNum());
+            "Committing instruction (seq %d)\n", inst_ptr->seqNum());
+
+    StaticInstPtr static_inst = inst_ptr->staticInst();
```

Comment:
? Is this needed?

Reply:
Well it was more to line up the parameter to DPRINTF, but I moved the call to inst_ptr->staticInst() later to right before I needed the variable I guess

Reply:
Oh I see. Ok.